pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md

```diff
@@ -0,0 +1,298 @@
+---
+$title: AMP for Email Fundamentals 
+$order: 1
+author: CrystalOnScript
+formats:
+  - email
+---
+
+If you're familiar with AMP, great news! AMP for Emails is just a subset of the AMP HTML library. If you're unfamiliar with AMP, also great news! This guide will give you everything you need to know to get started writing valid AMP Emails! 
+
+
+# Required Markup
+
+AMP Emails look like classic HTML emails, but with a few differences. Below is the minimum amount of markup required to make an email a valid AMP email.
+
+
+```html
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+</head>
+<body>
+  Hello, AMP4EMAIL world.
+</body>
+</html>
+```
+
+
+Email providers who support AMP Emails have set up security checks to ensure users get a delightful and safe experience. An email build with AMP must meet all requirements:
+
+
+
+*   Start with the `<!doctype html>` doctype. This is also standard for HTML.
+*   Contain a top-level a `<html amp4email>` tag, or an `<html ⚡4email>` tag if your email is extra cool. This identifies the document as an AMP Email so it can be treated as such.
+*   Define both `<head>` and `<body>` tags. This is optional in HTML, but AMP keeps things pristine!
+*   Include a `<meta charset="utf-8>` tag as the first child of the `<head>` tag. This identifies the encoding for the page.
+*   The AMP library is imported through a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag placed in the `<head>`. Without it, none of the awesome and dynamic functionality gained through AMP will work! As a best practice, this should be included as early as possible in the `<head>`, directly under the `<meta charset="utf-8">` tag.
+*   Initially hide the email content until the AMP library is loaded by placing the AMP for Email boilerplate in the `<head>`.
```

Comment:
Nit: this phrasing makes it sound like the boilerplate is optional. I'd go with:

"The AMP for Email boilerplate inside `<head>` ensures the email content stays hidden until the AMP library is loaded."